utils/python/CIME/preview_namelists.py

```diff
@@ -72,9 +72,12 @@ def preview_namelists(case, dryrun=False, casedir=None):
         cmd = os.path.join(config_dir, "buildnml")
         logger.info("Running %s"%cmd)
         if (logger.level == logging.DEBUG):
-            run_cmd("PREVIEW_NML=1 %s %s" % (cmd, caseroot))
+            rc, out, err = run_cmd("PREVIEW_NML=1 %s %s" % (cmd, caseroot), ok_to_fail=True)
```

Comment:
I need to be able to get output to the TestStatus.log file - so in case a user wants to add any DEBUG print statements, they could easily do so. At this point, any print statements that are added simply do no appear anywhere.